<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 8 About CytoTree | CytoTree Tutorial</title>
  <meta name="description" content="This is the tutorial of CytoTree R/Biocondcutor package." />
  <meta name="generator" content="bookdown 0.21.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 8 About CytoTree | CytoTree Tutorial" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is the tutorial of CytoTree R/Biocondcutor package." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 8 About CytoTree | CytoTree Tutorial" />
  
  <meta name="twitter:description" content="This is the tutorial of CytoTree R/Biocondcutor package." />
  

<meta name="author" content="Yuting Dai" />


<meta name="date" content="2020-11-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="structure-of-cytotree.html"/>

<script src="libs/header-attrs-2.5/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="https://github.com/JhuangLab/CytoTree"><center><b>CytoTree</b></center></a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="quick-start.html"><a href="quick-start.html"><i class="fa fa-check"></i><b>2</b> Quick Start</a>
<ul>
<li class="chapter" data-level="2.1" data-path="quick-start.html"><a href="quick-start.html#overview-of-workflow"><i class="fa fa-check"></i><b>2.1</b> Overview of Workflow</a></li>
<li class="chapter" data-level="2.2" data-path="quick-start.html"><a href="quick-start.html#installation"><i class="fa fa-check"></i><b>2.2</b> Installation</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="quick-start.html"><a href="quick-start.html#github"><i class="fa fa-check"></i><b>2.2.1</b> GitHub</a></li>
<li class="chapter" data-level="2.2.2" data-path="quick-start.html"><a href="quick-start.html#bioconductor"><i class="fa fa-check"></i><b>2.2.2</b> Bioconductor</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="quick-start.html"><a href="quick-start.html#quick-start-code"><i class="fa fa-check"></i><b>2.3</b> Quick-start code</a></li>
<li class="chapter" data-level="2.4" data-path="quick-start.html"><a href="quick-start.html#visualization"><i class="fa fa-check"></i><b>2.4</b> Visualization</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ti.html"><a href="ti.html"><i class="fa fa-check"></i><b>3</b> Trajectory Inference (TI)</a>
<ul>
<li class="chapter" data-level="3.1" data-path="ti.html"><a href="ti.html#preprocessing"><i class="fa fa-check"></i><b>3.1</b> Preprocessing</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="ti.html"><a href="ti.html#preprocessing-using-flowcore"><i class="fa fa-check"></i><b>3.1.1</b> Preprocessing using <code>flowCore</code></a></li>
<li class="chapter" data-level="3.1.2" data-path="ti.html"><a href="ti.html#preprocessing-using-cytotree"><i class="fa fa-check"></i><b>3.1.2</b> Preprocessing using <code>CytoTree</code></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="ti.html"><a href="ti.html#clustering"><i class="fa fa-check"></i><b>3.2</b> Clustering</a></li>
<li class="chapter" data-level="3.3" data-path="ti.html"><a href="ti.html#dimensionality-reduction"><i class="fa fa-check"></i><b>3.3</b> Dimensionality Reduction</a></li>
<li class="chapter" data-level="3.4" data-path="ti.html"><a href="ti.html#build-trajectory"><i class="fa fa-check"></i><b>3.4</b> Build Trajectory</a></li>
<li class="chapter" data-level="3.5" data-path="ti.html"><a href="ti.html#optimization"><i class="fa fa-check"></i><b>3.5</b> Optimization</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="analysis.html"><a href="analysis.html"><i class="fa fa-check"></i><b>4</b> Analysis of TI</a>
<ul>
<li class="chapter" data-level="4.1" data-path="analysis.html"><a href="analysis.html#pseudotime"><i class="fa fa-check"></i><b>4.1</b> Pseudotime</a></li>
<li class="chapter" data-level="4.2" data-path="analysis.html"><a href="analysis.html#intermediate-states"><i class="fa fa-check"></i><b>4.2</b> Intermediate States</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="time-course-data-use-case.html"><a href="time-course-data-use-case.html"><i class="fa fa-check"></i><b>5</b> Time-course data use case</a>
<ul>
<li class="chapter" data-level="5.1" data-path="time-course-data-use-case.html"><a href="time-course-data-use-case.html#step-1.-preprocessing"><i class="fa fa-check"></i><b>5.1</b> Step 1. Preprocessing</a></li>
<li class="chapter" data-level="5.2" data-path="time-course-data-use-case.html"><a href="time-course-data-use-case.html#step-2.-trajectory"><i class="fa fa-check"></i><b>5.2</b> Step 2. Trajectory</a></li>
<li class="chapter" data-level="5.3" data-path="time-course-data-use-case.html"><a href="time-course-data-use-case.html#step-3.-pseudotime"><i class="fa fa-check"></i><b>5.3</b> Step 3. Pseudotime</a></li>
<li class="chapter" data-level="5.4" data-path="time-course-data-use-case.html"><a href="time-course-data-use-case.html#step-4.-subset-cyt"><i class="fa fa-check"></i><b>5.4</b> Step 4. Subset CYT</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="advanced-usage.html"><a href="advanced-usage.html"><i class="fa fa-check"></i><b>6</b> Advanced Usage</a></li>
<li class="chapter" data-level="7" data-path="structure-of-cytotree.html"><a href="structure-of-cytotree.html"><i class="fa fa-check"></i><b>7</b> Structure of CytoTree</a></li>
<li class="chapter" data-level="8" data-path="about-cytotree.html"><a href="about-cytotree.html"><i class="fa fa-check"></i><b>8</b> About CytoTree</a>
<ul>
<li class="chapter" data-level="" data-path="about-cytotree.html"><a href="about-cytotree.html#bug-reports"><i class="fa fa-check"></i>Bug Reports</a></li>
<li class="chapter" data-level="" data-path="about-cytotree.html"><a href="about-cytotree.html#current-version"><i class="fa fa-check"></i>Current version</a></li>
<li class="chapter" data-level="" data-path="about-cytotree.html"><a href="about-cytotree.html#acknowledgment"><i class="fa fa-check"></i>Acknowledgment</a></li>
<li class="chapter" data-level="" data-path="about-cytotree.html"><a href="about-cytotree.html#session-information"><i class="fa fa-check"></i>Session Information</a></li>
<li class="chapter" data-level="" data-path="about-cytotree.html"><a href="about-cytotree.html#reference"><i class="fa fa-check"></i>Reference</a></li>
<li class="chapter" data-level="" data-path="about-cytotree.html"><a href="about-cytotree.html#about-the-author"><i class="fa fa-check"></i>About The Author</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">CytoTree Tutorial</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="about-cytotree" class="section level1" number="8">
<h1><span class="header-section-number">Chapter 8</span> About CytoTree</h1>
<div id="bug-reports" class="section level2 unnumbered">
<h2>Bug Reports</h2>
<p>If there is any error in installing or librarying the CytoTree package, please contact us via e-mail <a href="mailto:forlynna@sjtu.edu.cn" class="email">forlynna@sjtu.edu.cn</a>.</p>
</div>
<div id="current-version" class="section level2 unnumbered">
<h2>Current version</h2>
<p>Version: 1.0.2</p>
<p>Release Date: 2020-11-01</p>
</div>
<div id="acknowledgment" class="section level2 unnumbered">
<h2>Acknowledgment</h2>
<p>This work was supported by the National Key Research and Development Plan of China Grants [No. 2018YFA0107802], the National Natural Science Foundation of China (NSFC) General Program [No. 81570122, 81670094, 81770205, 81830007], the National Key Research and Development Program [No. 2016YFC0902800], and the Shanghai Municipal Education Commission-Gaofeng Clinical Medicine Grant Support [No. 20161303], Shanghai Collaborative Innovation Program on Regenerative Medicine and Stem Cell Research [No. 2019CXJQ01], Mayo Clinic Center for Individualized Medicine.</p>
<p>We thank the Center for High Performance Computing of Shanghai Jiao Tong University for providing computing support.</p>
<p>We thank Prof. Guangchuang Yu at Southern Medical University for his kind comments and advice of CytoTree.</p>
</div>
<div id="session-information" class="section level2 unnumbered">
<h2>Session Information</h2>
<div class="sourceCode" id="cb162"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb162-1"><a href="about-cytotree.html#cb162-1" aria-hidden="true"></a><span class="co"># Show session information</span></span>
<span id="cb162-2"><a href="about-cytotree.html#cb162-2" aria-hidden="true"></a><span class="kw">sessionInfo</span>()</span></code></pre></div>
<pre><code>## R version 4.0.2 (2020-06-22)
## Platform: x86_64-apple-darwin17.0 (64-bit)
## Running under: macOS Catalina 10.15.7
## 
## Matrix products: default
## BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
## [1] stringr_1.4.0   pheatmap_1.0.12 ggthemes_4.2.0  ggplot2_3.3.2   LSD_4.1-0       flowCore_2.0.1  CytoTree_1.0.2 
## [8] igraph_1.2.5   
## 
## loaded via a namespace (and not attached):
##   [1] reticulate_1.16             RUnit_0.4.32                tidyselect_1.1.0            RSQLite_2.2.0              
##   [5] AnnotationDbi_1.50.3        grid_4.0.2                  ranger_0.12.1               BiocParallel_1.22.0        
##   [9] Rtsne_0.15                  scatterpie_0.1.5            munsell_0.5.0               destiny_3.2.0              
##  [13] codetools_0.2-16            umap_0.2.6.0                withr_2.3.0                 colorspace_1.4-1           
##  [17] Biobase_2.48.0              highr_0.8                   knitr_1.30                  rstudioapi_0.11            
##  [21] stats4_4.0.2                SingleCellExperiment_1.10.1 robustbase_0.93-6           vcd_1.4-8                  
##  [25] VIM_6.0.0                   TTR_0.24.2                  labeling_0.3                GenomeInfoDbData_1.2.3     
##  [29] polyclip_1.10-0             bit64_4.0.5                 farver_2.0.3                flowWorkspace_4.0.6        
##  [33] vctrs_0.3.4                 generics_0.0.2              xfun_0.18                   R6_2.4.1                   
##  [37] GenomeInfoDb_1.24.2         RcppEigen_0.3.3.7.0         locfit_1.5-9.4              bitops_1.0-6               
##  [41] DelayedArray_0.14.1         scales_1.1.1                nnet_7.3-14                 gtable_0.3.0               
##  [45] sva_3.36.0                  RProtoBufLib_2.0.0          rlang_0.4.8                 genefilter_1.70.0          
##  [49] scatterplot3d_0.3-41        flowUtils_1.52.0            splines_4.0.2               hexbin_1.28.1              
##  [53] BiocManager_1.30.10         yaml_2.2.1                  abind_1.4-5                 RBGL_1.64.0                
##  [57] tools_4.0.2                 bookdown_0.21.2             ellipsis_0.3.1              RColorBrewer_1.1-2         
##  [61] proxy_0.4-24                BiocGenerics_0.34.0         Rcpp_1.0.5                  plyr_1.8.6                 
##  [65] base64enc_0.1-3             zlibbioc_1.34.0             purrr_0.3.4                 RCurl_1.98-1.2             
##  [69] FlowSOM_1.20.0              openssl_1.4.3               S4Vectors_0.26.1            zoo_1.8-8                  
##  [73] SummarizedExperiment_1.18.2 haven_2.3.1                 cluster_2.1.0               magrittr_1.5               
##  [77] ncdfFlow_2.34.0             data.table_1.13.0           RSpectra_0.16-0             openxlsx_4.2.2             
##  [81] gmodels_2.18.1              lmtest_0.9-38               RANN_2.6.1                  pcaMethods_1.80.0          
##  [85] matrixStats_0.56.0          hms_0.5.3                   evaluate_0.14               xtable_1.8-4               
##  [89] smoother_1.1                XML_3.99-0.5                rio_0.5.16                  jpeg_0.1-8.1               
##  [93] mclust_5.4.6                readxl_1.3.1                IRanges_2.22.2              gridExtra_2.3              
##  [97] ggcyto_1.16.0               compiler_4.0.2              tibble_3.0.3                crayon_1.3.4               
## [101] htmltools_0.5.0             mgcv_1.8-33                 corpcor_1.6.9               tidyr_1.1.2                
## [105] RcppParallel_5.0.2          DBI_1.1.0                   tweenr_1.0.1                MASS_7.3-53                
## [109] boot_1.3-25                 Matrix_1.2-18               car_3.0-9                   gdata_2.18.0               
## [113] parallel_4.0.2              GenomicRanges_1.40.0        forcats_0.5.0               pkgconfig_2.0.3            
## [117] rvcheck_0.1.8               prettydoc_0.4.0             foreign_0.8-80              laeken_0.5.1               
## [121] sp_1.4-2                    xml2_1.3.2                  annotate_1.66.0             XVector_0.28.0             
## [125] digest_0.6.27               tsne_0.1-3                  ConsensusClusterPlus_1.52.0 graph_1.66.0               
## [129] rmarkdown_2.5               cellranger_1.1.0            edgeR_3.30.3                curl_4.3                   
## [133] gtools_3.8.2                ggplot.multistats_1.0.0     lifecycle_0.2.0             nlme_3.1-149               
## [137] jsonlite_1.7.1              carData_3.0-4               BiocNeighbors_1.6.0         askpass_1.1                
## [141] limma_3.44.3                pillar_1.4.6                lattice_0.20-41             DEoptimR_1.0-8             
## [145] survival_3.2-3              glue_1.4.2                  xts_0.12.1                  zip_2.1.1                  
## [149] png_0.1-7                   bit_4.0.4                   Rgraphviz_2.32.0            ggforce_0.3.2              
## [153] class_7.3-17                stringi_1.5.3               blob_1.2.1                  RcppHNSW_0.3.0             
## [157] CytoML_2.0.5                latticeExtra_0.6-29         memoise_1.1.0               dplyr_1.0.2                
## [161] cytolib_2.0.3               knn.covertree_1.0           irlba_2.3.3                 e1071_1.7-3</code></pre>
</div>
<div id="reference" class="section level2 unnumbered">
<h2>Reference</h2>
<ol style="list-style-type: decimal">
<li>Hahne F, Arlt D, Sauermann M, Majety M, Poustka A, Wiemann S, Huber W: Statistical methods and software for the analysis of highthroughput reverse genetic assays using flow cytometry readouts. Genome Biol 2006, 7:R77.</li>
<li>Olsen LR, Leipold MD, Pedersen CB, Maecker HT: The anatomy of single cell mass cytometry data. Cytometry A 2019, 95:156-172.</li>
<li>Butler A, Hoffman P, Smibert P, Papalexi E, Satija R: Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nat Biotechnol 2018, 36:411-420.</li>
<li>Trapnell C, Cacchiarelli D, Grimsby J, Pokharel P, Li S, Morse M, Lennon NJ, Livak KJ, Mikkelsen TS, Rinn JL: The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Nat Biotechnol 2014, 32:381-386.</li>
<li>Kiselev VY, Yiu A, Hemberg M: scmap: projection of single-cell RNA-seq data across data sets. Nat Methods 2018, 15:359-362.</li>
<li>Amir el AD, Davis KL, Tadmor MD, Simonds EF, Levine JH, Bendall SC, Shenfeld DK, Krishnaswamy S, Nolan GP, Pe’er D: viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia. Nat Biotechnol 2013, 31:545-552.</li>
<li>Haghverdi L, Buettner F, Theis FJ: Diffusion maps for high-dimensional single-cell analysis of differentiation data. Bioinformatics 2015, 31:2989-2998.</li>
<li>Becht E, McInnes L, Healy J, Dutertre CA, Kwok IWH, Ng LG, Ginhoux F, Newell EW: Dimensionality reduction for visualizing single-cell data using UMAP. Nat Biotechnol 2018.</li>
<li>Wang L, Hoffman RA: Standardization, Calibration, and Control in Flow Cytometry. Curr Protoc Cytom 2017, 79:1 3 1-1 3 27.</li>
<li>Hahne F, LeMeur N, Brinkman RR, Ellis B, Haaland P, Sarkar D, Spidlen J, Strain E, Gentleman R: flowCore: a Bioconductor package for high throughput flow cytometry. BMC Bioinformatics 2009, 10:106.</li>
<li>Sarkar D, Le Meur N, Gentleman R: Using flowViz to visualize flow cytometry data. Bioinformatics 2008, 24:878-879.</li>
<li>Van Gassen S, Callebaut B, Van Helden MJ, Lambrecht BN, Demeester P, Dhaene T, Saeys Y: FlowSOM: Using self-organizing maps for visualization and interpretation of cytometry data. Cytometry A 2015, 87:636-645.</li>
<li>Qiu P, Simonds EF, Bendall SC, Gibbs KD, Jr., Bruggner RV, Linderman MD, Sachs K, Nolan GP, Plevritis SK: Extracting a cellular hierarchy from high-dimensional cytometry data with SPADE. Nat Biotechnol 2011, 29:886-891.</li>
<li>Chen H, Lau MC, Wong MT, Newell EW, Poidinger M, Chen J: Cytofkit: A Bioconductor Package for an Integrated Mass Cytometry Data Analysis Pipeline. PLoS Comput Biol 2016, 12:e1005112.</li>
<li>Chattopadhyay PK, Winters AF, Lomas WE, 3rd, Laino AS, Woods DM: High-Parameter Single-Cell Analysis. Annu Rev Anal Chem (Palo Alto Calif) 2019, 12:411-430.</li>
<li>Bendall SC, Davis KL, Amir el AD, Tadmor MD, Simonds EF, Chen TJ, Shenfeld DK, Nolan GP, Pe’er D: Single-cell trajectory detection uncovers progression and regulatory coordination in human B cell development. Cell 2014, 157:714-725.</li>
<li>Nowicka M, Krieg C, Crowell HL, Weber LM, Hartmann FJ, Guglietta S, Becher B, Levesque MP, Robinson MD: CyTOF workflow: differential discovery in high-throughput high-dimensional cytometry datasets. F1000Res 2017, 6:748.</li>
<li>Bendall SC, Simonds EF, Qiu P, Amir el AD, Krutzik PO, Finck R, Bruggner RV, Melamed R, Trejo A, Ornatsky OI, et al: Single-cell mass cytometry of differential immune and drug responses across a human hematopoietic continuum. Science 2011, 332:687-696.</li>
<li>Herring CA, Banerjee A, McKinley ET, Simmons AJ, Ping J, Roland JT, Franklin JL, Liu Q, Gerdes MJ, Coffey RJ, Lau KS: Unsupervised Trajectory Analysis of Single-Cell RNA-Seq and Imaging Data Reveals Alternative Tuft Cell Origins in the Gut. Cell Syst 2018, 6:37-51 e39.</li>
<li>Spidlen J, Breuer K, Rosenberg C, Kotecha N, Brinkman RR: FlowRepository: a resource of annotated flow cytometry datasets associated with peer-reviewed publications. Cytometry A 2012, 81:727-731.</li>
<li>Doulatov S, Notta F, Laurenti E, Dick JE: Hematopoiesis: a human perspective. Cell Stem Cell 2012, 10:120-136.</li>
<li>Laurenti E, Gottgens B: From haematopoietic stem cells to complex differentiation landscapes. Nature 2018, 553:418-426.</li>
<li>Wang C, Tang X, Sun X, Miao Z, Lv Y, Yang Y, Zhang H, Zhang P, Liu Y, Du L, et al: TGFbeta inhibition enhances the generation of hematopoietic progenitors from human ES cell-derived hemogenic endothelial cells using a stepwise strategy. Cell Res 2012, 22:194-207.</li>
<li>Notta F, Doulatov S, Laurenti E, Poeppl A, Jurisica I, Dick JE: Isolation of single human hematopoietic stem cells capable of long-term multilineage engraftment. Science 2011, 333:218-221.</li>
<li>Hu Z, Jujjavarapu C, Hughey JJ, Andorf S, Lee HC, Gherardini PF, Spitzer MH, Thomas CG, Campbell J, Dunn P, et al: MetaCyto: A Tool for Automated Meta-analysis of Mass and Flow Cytometry Data. Cell Rep 2018, 24:1377-1388.</li>
<li>Finak G, Frelinger J, Jiang W, Newell EW, Ramey J, Davis MM, Kalams SA, De Rosa SC, Gottardo R: OpenCyto: an open source infrastructure for scalable, robust, reproducible, and automated, end-to-end flow cytometry data analysis. PLoS Comput Biol 2014, 10:e1003806.</li>
<li>Aghaeepour N, Nikolic R, Hoos HH, Brinkman RR: Rapid cell population identification in flow cytometry data. Cytometry A 2011, 79:6-13.</li>
<li>Schubert E, Rousseeuw PJ: Faster k-Medoids Clustering: Improving the PAM, CLARA, and CLARANS Algorithms. 2018.</li>
<li>Scrucca L, Fop M, Murphy TB, Raftery AE: mclust 5: Clustering, Classification and Density Estimation Using Gaussian Finite Mixture Models. R j 2016, 8:289-317.</li>
<li>Leek JT, Johnson WE, Parker HS, Jaffe AE, Storey JD: The sva package for removing batch effects and other unwanted variation in high-throughput experiments. Bioinformatics 2012, 28:882-883.</li>
<li>Saelens W, Cannoodt R, Todorov H, Saeys Y: A comparison of single-cell trajectory inference methods. Nat Biotechnol 2019.</li>
<li>Kotecha N, Krutzik PO, Irish JM: Web-based analysis and publication of flow cytometry experiments. Curr Protoc Cytom 2010, Chapter 10:Unit10 17.</li>
<li>Finak G, Jiang M: FlowWorkspace: Infrastructure for Representing and Interacting with the Gated Cytometry. 2011.</li>
<li>Van P, Jiang W, Gottardo R, Finak G: ggCyto: next generation open-source visualization software for cytometry. Bioinformatics 2018, 34:3951-3953.</li>
<li>Ito K, Murphy D: Application of ggplot2 to Pharmacometric Graphics. CPT Pharmacometrics Syst Pharmacol 2013, 2:e79.</li>
<li>Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, Smyth GK: limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res 2015, 43:e47.</li>
<li>Cao J, Spielmann M, Qiu X, Huang X, Ibrahim DM, Hill AJ, Zhang F, Mundlos S, Christiansen L, Steemers FJ, et al: The single-cell transcriptional landscape of mammalian organogenesis. Nature 2019, 566:496-502.</li>
<li>Wang X: A Fast Exact k-Nearest Neighbors Algorithm for High Dimensional Search Using k-Means Clustering and Triangle Inequality. Proc Int Jt Conf Neural Netw 2012, 43:2351-2358.</li>
</ol>
</div>
<div id="about-the-author" class="section level2 unnumbered">
<h2>About The Author</h2>
<p>The author <a href="https://www.researchgate.net/profile/Yuting_Dai3">Yuting Dai</a> is a Ph.D. student at Shanghai Jiao Tong University. In 2016, she graduated from Shanghai Jiao Tong University and got a Bachelor Degree in Bioinformatics. Then she continued to study for a Ph.D. at Shanghai Institute of Hematology, Shanghai Jiao Tong University. The main research directions of her are bioinformatic tools or workflows development on multi-omics data in cancer, and the details as follow:</p>
<ol style="list-style-type: decimal">
<li><p>Sequencing data analysis in cancer, especially leukemia, e.g. whole-genome sequencing, whole-exome sequencing, RNA sequencing, single-cell RNA sequencing, ChIP-seq.</p></li>
<li><p>Flow and mass cytometry data analysis and software development.</p></li>
<li><p>Visualization of multidimensional data.</p></li>
</ol>
<p>She has participated in several projects and here is the publication (# represents the co-first author, and * represents the corresponding author):</p>
<p>[1] Lu Jiang<sup>#</sup>, Xue-Ping Li<sup>#</sup>, <strong>Yu-Ting Dai<sup>#</sup></strong>, Bing Chen, Xiang-Qin Weng, Shu-Min Xiong, Min Zhang, Jin-Yan Huang*, Zhu Chen*, Sai-Juan Chen*. <a href="https://pubmed.ncbi.nlm.nih.gov/32747558/">Multidimensional study of the heterogeneity of leukemia cells in t(8;21) acute myelogenous leukemia identifies the subtype with poor outcome</a>. <strong><em>PNAS</em></strong>, 2020, 117(33):20117-20126.</p>
<p>[2] Jie Xiong<sup>#</sup>, Bo-Wen Cui<sup>#</sup>, Nan Wang<sup>#</sup>, <strong>Yu-Ting Dai<sup>#</sup></strong>, Hao Zhang#, Chao-Fu Wang#, Hui-Juan Zhong, Shu Cheng, Bin-Shen Ou-Yang, Yu Hu, Xi Zhang, Bin Xu, Wen-Bin Qian, Rong Tao, Feng Yan, Jian-Da Hu, Ming Hou, Xue-Jun Ma, Xin Wang, Yuan-Hua Liu, Zun-Min Zhu, Xiao-Bin Huang, Li Liu, Chong-Yang Wu, Li Huang, Yun-Feng Shen, Rui-Bin Huang, Jing-Yan Xu, Chun Wang, De-Pei Wu, Li Yu, Jian-Feng Li, Peng-Peng Xu, Li Wang, Jin-Yan Huang*, Sai-Juan Chen*, Wei-Li Zhao*. <a href="https://pubmed.ncbi.nlm.nih.gov/32183952/">Genomic and Transcriptomic Characterization of Natural Killer T Cell Lymphoma</a>. <strong><em>Cancer Cell</em></strong>, 2020, 37(3):403-419.</p>
<p>[3] Yi Zhou<sup>#</sup>, Xingli Zhu<sup>#</sup>, <strong>Yuting Dai<sup>#</sup></strong>, Shumin Xiong, Chuijin Wei, Pei Yu, Yuewen Tang, Liang Wu, Jianfeng Li, Dan Liu, Yanlin Wang, Zhu Chen, Sai-Juan Chen*, Jinyan Huang*, Lin Cheng*. <a href="https://pubmed.ncbi.nlm.nih.gov/31921559/">Chemical cocktail induces hematopoietic reprogramming and expands hematopoietic stem/progenitor cells</a>. <strong><em>Advanced Science</em></strong>, 2019. 7(1):1901785</p>
<p>[4] Jian-Feng Li<sup>#</sup>, <strong>Yu-Ting Dai<sup>#</sup></strong>, Henrik Lilljebjörn<sup>#</sup>, Shu-Hong Shen, Bo-Wen Cui, Ling Bai, Yuan-Fang Liu, Mao-Xiang Qian, Yasuo Kubota, Hitoshi Kiyoi, Itaru Matsumura, Yasushi Miyazaki, Linda Olsson, Ah Moy Tan, Hany Ariffin, Jing Chen, Junko Takita, Takahiko Yasuda, Hiroyuki Mano, Bertil Johansson, Jun J. Yang, Allen Yeoh Eng Juh, Fumihiko Hayakawa, Zhu Chen*, Ching-Hon Pui*, Thoas Fioretos*, Sai-Juan Chen*, Jin-Yan Huang*. <a href="https://pubmed.ncbi.nlm.nih.gov/30487223/">The transcriptional landscape of B-cell precursor acute lymphoblastic leukemia based on an international study of 1,223 cases</a>. <strong><em>PNAS</em></strong>, 2018. 115(50):E11711-E11720.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="structure-of-cytotree.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
